dev-libs/ffcall/files/ffcall-2.4-slibtool.patch

```diff
@@ -0,0 +1,85 @@
+Bug: https://bugs.gentoo.org/776976
```

Comment:
Is this upstream somewhre?

Reply:
I tried to send a patch to the mailing list, but I don't see it yet.

https://lists.gnu.org/archive/html/libffcall/

Reply:
I see it now.

https://lists.gnu.org/archive/html/libffcall/2022-07/msg00000.html

Reply:
Thanks, I'll throw the link in.

Reply:
oh, you already did :)